from_mwt_ds/DataScience/vw_executor/artifacts.py

```diff
@@ -101,3 +106,58 @@ def metrics(self):
         if not self._processed:
             self._process()
         return self._metrics
+
+class Predictions(Artifact):
+    def __init__(self, path):
+        super().__init__(path)
+
+    @property
+    def cb(self):
```

Comment:
add one for cats maybe? preds look like `action:pdf_value`, so similar to `cb` but with only one prediction per line. Now that I think about it, is there a corner case for cb where we only have one action?

Reply:
added cats one
added corner cases to tests